patches/nginx.patch

```diff
@@ -198,11 +208,11 @@ index 7b0417e4..ff16cfc8 100644
              return NGX_ERROR;
          }
 diff --git a/src/event/ngx_event_openssl.h b/src/event/ngx_event_openssl.h
-index c9e86d9c..d037b956 100644
+index 61da0c5d..1c6d99f4 100644
 --- a/src/event/ngx_event_openssl.h
 +++ b/src/event/ngx_event_openssl.h
-@@ -119,6 +119,11 @@ struct ngx_ssl_connection_s {
-     unsigned                    in_ocsp:1;
+@@ -99,6 +99,11 @@ struct ngx_ssl_connection_s {
+     unsigned                    in_early:1;
```

Comment:
its ok?

Reply:
@paragor  still testing.  revert ed this merge.

Please refer: https://github.com/phuslu/nginx-ssl-fingerprint/pull/23

Reply:
@paragor  Can you join the slack channel

https://app.slack.com/client/TD15QFV34/C03R9MLQLJ3/activity

Reply:
@deancn link not work
<img width="748" alt="image" src="https://user-images.githubusercontent.com/31911894/182774708-8d1e0dcc-1fd1-410b-a376-e9550f3ca6ae.png">

Reply:
@paragor  Can you send an email to me,  let me invite you.

dean@airdb.com

Reply:
send

Reply:
Haven't received it.  Please send it again.

Reply:
novikov46en@gmail.com